google-http-client/src/main/java/com/google/api/client/testing/http/apache/MockHttpClient.java

```diff
@@ -50,13 +50,14 @@
  * @since 1.14
  * @author Yaniv Inbar
  */
-@Beta
+@Deprecated
```

Comment:
Let's give more details about the deprecation - e.g. can it not be used any more with this library because of the upgrade of the Apache http client version?

Reply:
clirr (a Maven plugin) kept failing other PRs because this file was moved all together. I was going to deprecate it in this version and remove it completely in the future.